VTEX - VTEX Do API.json

```diff
@@ -197,17 +296,24 @@
                     {
                         "name": "noteId",
                         "in": "path",
-                        "description": "Note's ID.",
+                        "description": "Note ID.",
                         "required": true,
                         "style": "simple",
                         "schema": {
                             "type": "string",
-                            "example": "654321cba",
-                            "description": "Note's ID."
+                            "example": "654321cba"
                         }
                     },
                     {
-                        "$ref": "#/components/parameters/reason"
+                        "name": "reason",
+                        "in": "query",
+                        "description": "This parameter is relevant only for PII-compliant accounts. When sending requests to this endpoint, PII-compliant accounts can use this parameter to declare the reason for requesting unmasked data. Otherwise, this endpoint will return masked PII data.",
```

Comment:
See comment above

Reply:
The text was already like this in the documentation, but before the field was declared in "components". Since it appeared only twice, I removed the mention from "components" to the usual parameters in "path". I did it thinking about not overusing "$ref" and having a more organized code.

After your comment, I checked it with Julia and she suggested changing it, so I did, and I agree it is better!